src/util/Recoil_stableStringify.js

```diff
@@ -126,22 +126,28 @@ function stringify(x: mixed, opt: Options, key?: string): string {
 // * We could support BigInt, but Flow doesn't seem to like it.
 // See Recoil_stableStringify-test.js for examples
 function stableStringify(x: mixed, opt?: Options): string {
+  const shouldEvalPerformance = __DEV__ && typeof window !== 'undefined';
```

Comment:
Sorry, for the code builds to work fully they require `if (__DEV__)` checks to be their own statements.

Reply:
oh ok, np. How does my latest push look?